.vscode/launch.json

```diff
@@ -31,24 +31,24 @@
             ],
         },
         {
-            "name": "Launch server (sqlite file)",
+            "name": "Launch server (sqlite file) - for integration tests",
```

Comment:
please don't bork this one but create another one instead — otherwise we won't have any developing against sqlite and restarting the dev container

Reply:
will do